src/Illuminate/Database/Eloquent/Builder.php

```diff
@@ -1038,8 +1038,10 @@ public function upsert(array $values, $uniqueBy, $update = null)
      */
     public function increment($column, $amount = 1, array $extra = [])
     {
-        return $this->toBase()->increment(
-            $column, $amount, $this->addUpdatedAtColumn($extra)
+        return $this->update(
+            $this->toBase()->createIncrementStatement(
```

Comment:
```suggestion
            $this->createIncrementStatement(
```
You already have the `toBase()` call inside of `createIncrementStatement()`

Reply:
i've moved 2 functions to the `$passthru` variable so they can be called directly. However I've kept the `toBase()` call since I don't see it done for any of the other passthru functions.